src/vulkan/visualizer.rs

```diff
@@ -275,4 +280,121 @@ impl AllocatorVisualizer {
             self.render_memory_block_windows(ui, allocator);
         }
     }
+    
+    pub fn render_breakdown(&mut self, allocator: &Allocator, ui: &imgui::Ui<'_>, opened: Option<&mut bool>) {
+        let mut allocation_report = vec![];
+        let mut total_size_in_bytes = 0;
+
+        if let Some(true) = opened {
+            let lowercase_needle = &self.breakdown_filter.to_lowercase();
+            for memory_type in &allocator.memory_types {
+                for block in &memory_type.memory_blocks {
+                    if let Some(block) = block {
+                        for report in block.sub_allocator.report_allocations() {
+                            if self.breakdown_filter.is_empty() || report.name.to_lowercase().contains(lowercase_needle) {
+                                allocation_report.push(report);
+                            }
+                        }
+                    }
+                }
+            }
+
+            total_size_in_bytes = allocation_report.iter().map(|report| report.size).sum();
+        }
+
+        let suffix = ["B", "KB", "MB", "GB", "TB"];
```

Comment:
Technically this should be KiB, MiB, GiB, etc
```suggestion
        let suffix = ["B", "KiB", "MiB", "GiB", "TiB"];
```